lib/hammer_cli_foreman_discovery/discovery.rb

```diff
@@ -24,7 +25,27 @@ class InfoCommand < HammerCLIForeman::InfoCommand
       output ListCommand.output_definition do
         field :ip, _('Ip')
         field :model, _('Model')
-        field :facts_hash, _('Facts')
```

Comment:
I removed this because facts now have a command with a nicer output.
Should the facts still appear in the info output?
for managed hosts the facts aren't shown in info.